_elements/tables.md

```diff
@@ -5,8 +5,95 @@ title: Tables
 ---
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Tables_UI_v1.png">
+
+  <h3>Bordered Table</h3>
+
+  <table class="usa-table-bordered">
```

Comment:
What if a table has not class (neither bordered or unbordered)? Does it still look OK?